src/components/GamePage.vue

```diff
@@ -125,69 +125,75 @@ keys["Alt"] = "Alt";
 keys["Meta"] = "Meta";
 keys[" "] = "space";
 
-
+//使いわますため（次のインデックスにするためのやつ）
+function substfunc(ind:num) {
```

Comment:
今の関数名では 文字列の切り出し処理 としかわかりません
setNextIndexCode など、何の処理をしているかわかる関数名にしましょう。(この関数名がbestだと思いませんが..)

- 引数の名称は `i`もしくは `index`で
- num型ではなくnumber型です
```suggestion
//次のインデックスに変更
function substfunc(index:number) {
```